external/anomaly/ote_anomalib/exportable_code/anomaly_segmentation.py

```diff
@@ -0,0 +1,43 @@
+"""Wrapper for Open Model Zoo for Anomaly Segmentation tasks."""
+
+# Copyright (C) 2021-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from typing import Any, Dict
+
+import cv2
+import numpy as np
+
+from .base import AnomalyBase
+
+
+class AnomalySegmentation(AnomalyBase):
+    """Wrapper for anomaly segmentation task."""
+
+    __model__ = "anomaly_segmentation"
+
+    def postprocess(self, outputs: Dict[str, np.ndarray], meta: Dict[str, Any]) -> np.ndarray:
+        """Resize the outputs of the model to original image size.
+
+        Args:
+            outputs (Dict[str, np.ndarray]): Raw outputs of the model after ``infer_sync`` is called.
+            meta (Dict[str, Any]): Metadata which contains values such as threshold, original image size.
+
+        Returns:
+            np.ndarray: Segmentation Mask
+        """
+        anomaly_map: np.ndarray = outputs[self.output_blob_name].squeeze()
+
+        meta["image_threshold"] = self.image_threshold  # pylint: disable=no-member
+        meta["min"] = self.min  # pylint: disable=no-member
+        meta["max"] = self.max  # pylint: disable=no-member
+        meta["threshold"] = self.threshold  # pylint: disable=no-member
+
+        anomaly_map = self._normalize(anomaly_map, meta["image_threshold"], meta["min"], meta["max"])
```

Comment:
We need to use the pixel threshold here instead of image threshold. Though I think it is not yet stored in the output model. I will add it to the output model in https://github.com/openvinotoolkit/training_extensions/pull/913. Once it's added you could update this PR.

Reply:
@ashwinvaidya17 I've pushed the changes related to pixel threshold to https://github.com/openvinotoolkit/training_extensions/pull/913. You should be able to access the pixel threshold value from the meta data.

Reply:
I'll make the changes.